backends/magma/ceed-magma.h

```diff
@@ -139,7 +258,14 @@ CEED_INTERN {
 // comment the line below to use the default magma_is_devptr function
 #define magma_is_devptr magma_isdevptr
 
+// if magma and cuda/ref are using the null stream, then ceed_magma_queue_sync
+// should do nothing
+#define ceed_magma_queue_sync(...)
+//#define ceed_magma_queue_sync    magma_queue_sync
```

Comment:
Also dead

Reply:
Done in 195ca8ef85eeb68ebbdc2c75c6f28c876ee1d0a1